dts/riscv/sophgo/cv180x.dtsi

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2023-2024 Chen Xingyu <hi@xingrz.me>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+
+#include <sophgo/cv18xx-c906-common.dtsi>
+
+/ {
+	soc {
+		compatible = "sophgo,cv180x";
```

Comment:
simple-bus?

Reply:
Do you mean `compatible = "sophgo,cv180x", "simple-bus";`?

Reply:
yes